src/components/datepicker/readme.md

```diff
@@ -1,6 +1,16 @@
-# fw-datepicker
+# Datepicker (fw-datepicker)
+fw-datepicker displays calender with prepopulated month and year and enables to picking a date or date range. The calender displaying based mode attribute to select a single date or range of dates.
 
 
+## Usage
+
+``` html live
+<fw-datepicker></fw-datepicker>
+<fw-datepicker mode="range"></fw-datepicker>
+<fw-datepicker value="22/05/2020" date-format="MM/DD/YYYY"></fw-datepicker>
+<fw-datepicker mode="range" min-date="10-05-2020" max-date="10-07-2020"></fw-datepicker>
+```
+
 
 <!-- Auto Generated Below -->
```

Comment:
Please use the content in the description column of the [date-picker](https://docs.google.com/document/d/16tASosMY4o9ZTV4J-jSGt0bznc_5Ylc3bi4KmlZHe2s/edit#heading=h.8d0twk6lit8y) document to update auto-generated content.